dashboard/test/fixtures/test-serialize-seeding-json.script_json

```diff
@@ -101,10 +101,111 @@
       }
     }
   ],
+  "lesson_activities": [
+    {
+      "key": "test-serialize-seeding-json-lg-1-lesson-1-activity-1",
+      "position": 1,
```

Comment:
it looks like position is always 1. is that correct?

Reply:
I took a shortcut in `create_script_tree` and only made there be 1 LessonActivity per Lesson, and only 1 ActivitySection per LessonActivity. So, this is correct in one sense (working as I currently intended), but not in another (Winter would probably say that there should always be at least 2 things in a list in tests). Since this PR is getting so big, and nothing is relying on this code yet, how about I work on increasing the number of these from 1 to 2 in a separate PR? I'll add this as a note in [PLAT-609](https://codedotorg.atlassian.net/browse/PLAT-609).

Reply:
Waiting until a later PR sounds fine to me